decred_wallet/Features/Security/PinInputView.swift

```diff
@@ -56,14 +60,14 @@ class PinInputView: UIView {
     
     func drawPinLabel() {
         let pinDigitsCount = String(pin.count)
-        
-        let pinLabel = UILabel()
-        pinLabel.text = String(pinDigitsCount)
+        let pinLabel = UILabel(frame: self.bounds) // Set the label bounds to resolve any ambiguity
+        pinLabel.translatesAutoresizingMaskIntoConstraints = false
+        pinLabel.text = pinDigitsCount
         pinLabel.textAlignment = .center
         pinLabel.textColor = #colorLiteral(red: 0.2537069321, green: 0.8615272641, blue: 0.7028611302, alpha: 1)
         pinLabel.font = pinLabel.font.withSize(25)
-        
-        self.addSubview(pinLabel)
+        // View is set, add it to parent
+        addSubview(pinLabel)
```

Comment:
- Is this different from `self.addSubView`?
- Also, the comment is not necessary, it's obvious the label is set (it was created just above) and that we're adding it to the parent.

Reply:
> no difference at all. i was just keeping it short
> fixed in commit d097f900b6d316d6272559d520696d0fec66710a